lmdeploy/cli/utils.py

```diff
@@ -236,15 +317,22 @@ def cache_max_entry_count(parser):
 
     @staticmethod
     def adapters(parser):
+        """Add argument adapters to parser."""
+
         return parser.add_argument(
             '--adapters',
+            nargs='*',
+            type=str,
             default=None,
-            action=DictAction,
-            help='Used key-values pairs in xxx=yyy format'
-            ' to set the path lora adapter')
+            help='Used to set path(s) of lora adapter(s). One can input '
+            'key-value pairs in xxx=yyy format for multiple lora '
```

Comment:
any separator between key-value pairs?

Reply:
the empty space  is the separator for all cli.